core/tauri-macros/src/command/wrapper.rs

```diff
@@ -211,7 +228,17 @@ pub fn wrapper(attributes: TokenStream, item: TokenStream) -> TokenStream {
           use #root::command::private::*;
           // prevent warnings when the body is a `compile_error!` or if the command has no arguments
           #[allow(unused_variables)]
-          let #root::ipc::Invoke { message: #message, resolver: #resolver } = $invoke;
+          let #root::ipc::Invoke { id: #id, message: #message, resolver: #resolver } = $invoke;
+
+          let _span = tracing::debug_span!(
+            "ipc::request::handler",
```

Comment:
Isn't this the top-level span for ipc requests? If so I would just use `ipc::request` for this and keep the `ipc::request::handler` for what's called `ipc::request::run` right now

```suggestion
            "ipc::request",
```

Reply:
it's not the top level, there's more logic in window itself before it gets here (plus right now the top-level span is in wry).